extensions/wikia/AssetsManager/config.php

```diff
@@ -109,15 +109,17 @@
 		'//extensions/wikia/AdEngine/js/utils/eventDispatcher.js',
 		'//extensions/wikia/AdEngine/js/utils/hooks.js',
 		'//extensions/wikia/AdEngine/js/utils/math.js',
+		'//extensions/wikia/AdEngine/js/video/player/playwire/playwirePlayerFactory.js',
+		'//extensions/wikia/AdEngine/js/video/player/playwire/playwire.js',
+		'#group_adengine2_porvata_js',
 		'//extensions/wikia/AdEngine/js/video/player/ui/closeButton.js',
-		'//extensions/wikia/AdEngine/js/video/playwire.js',
-		'//extensions/wikia/AdEngine/js/video/googleIma.js',
-		'//extensions/wikia/AdEngine/js/video/googleImaAdStatus.js',
+		'//extensions/wikia/AdEngine/js/video/player/ui/pauseOverlay.js',
```

Comment:
What do you think of grouping those assets as well, just like `adengine_porvata_js` below? I wish we could lazy load those assets instead of putting everything in `adengine2_desktop_js`...